utils/model.py

```diff
@@ -303,6 +323,12 @@ def __llama_cpp_model_stream(self, model: "Llama", prompt: str, generation_confi
         for output in model(prompt, max_tokens=generation_config.__dict__['max_new_tokens'], temperature=generation_config.__dict__['temperature'], top_p=generation_config.__dict__['top_p'], repeat_penalty=generation_config.__dict__['repetition_penalty'], frequency_penalty=generation_config.__dict__['frequency_penalty'], stream=True):
             yield output['choices'][0]['text'], output['choices'][0]['finish_reason']
 
+    def __vllm_model_stream(self, model: "vLLM", prompt: str, generation_config: GenerationConfig):
+        logger.debug(f"prompt is: {prompt}")
+        sampling_params = SamplingParams(max_tokens=generation_config.__dict__['max_new_tokens'], temperature=generation_config.__dict__['temperature'], top_p=generation_config.__dict__['top_p'], repeat_penalty=generation_config.__dict__['repetition_penalty'], frequency_penalty=generation_config.__dict__['frequency_penalty'])
```

Comment:
```python
max_new_tokens = generation_config.__dict__['max_new_tokens']
temperature = generation_config.__dict__['temperature']
top_p = generation_config.__dict__['top_p']
repetition_penalty = generation_config.__dict__['repetition_penalty']
frequency_penalty = generation_config.__dict__['frequency_penalty']

sampling_params = SamplingParams(
    max_tokens=max_new_tokens, 
    temperature=temperature, 
    top_p=top_p, 
    repeat_penalty=repetition_penalty,  
    frequency_penalty=frequency_penalty,
)
# or
sampling_params = SamplingParams(
    max_tokens=generation_config.__dict__['max_new_tokens'], 
    temperature=generation_config.__dict__['temperature'], 
    top_p=generation_config.__dict__['top_p'], 
    repeat_penalty=generation_config.__dict__['repetition_penalty'],  
    frequency_penalty=generation_config.__dict__['frequency_penalty'],
)
```
please do not crowd all parameters in one line.

Reply:
My fault, the `__llama_cpp_model_stream` and `__llama_cpp_model` function are so written, I'll correct it soon.